packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -106,7 +106,13 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
             files.push(inputFile)
 
             for (let file of files) {
-              let changedTime = fs.statSync(file, { throwIfNoEntry: false })?.mtimeMs ?? null
+              let changedTime: null | number
+              try {
+                changedTime = fs.statSync(file, { throwIfNoEntry: false })?.mtimeMs ?? null
+              } catch {
+                changedTime = null
+              }
```

Comment:
I'd remove the `throwIfNoEntry` bit since this try catch will result in identical behavior in that case too

Reply:
yeah but now I wonder if this really is the reason for the throw at all. Guess I do need to work a bit more on getting a proper repro first.

Reply:
Dude I’m stupid I updated the wrong file. I wanted to add the try/catch in `packages/@tailwindcss-cli/src/commands/build/index.ts`. Awkward